pkg/v5/configmap/configmap.go

```diff
@@ -4,24 +4,31 @@ import (
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 	corev1 "k8s.io/api/core/v1"
+
+	"github.com/giantswarm/cluster-operator/pkg/v5/guestcluster"
 )
 
 // Config represents the configuration used to create a new configmap service.
 type Config struct {
+	Guest  guestcluster.Interface
```

Comment:
guestcluster service handles getting a K8s client for the guest cluster.

Reply:
I am thinking to extract that soon into its own repo because we do this in quite a lot of places.

Reply:
Good point. Converting to a library would be useful. We could use it in the e2e tests and probably some more places.